tests/cython/CMakeLists.txt

```diff
@@ -22,6 +22,22 @@ IF (HAVE_SERVER)
 ENDIF (HAVE_SERVER)
 
 
+# The PythonEvalUTest is derived from the CogServer class, and so 
+# cannot be tested unless the cogserver is actually being built!
+IF (HAVE_SERVER)
```

Comment:
Oh,. never mind, I see you are using the server.  Many of the test cases use the server when they don't really need to -- its enough to create a bare atomspace, the rest of the server is typically un-needed.